pkg/META

```diff
@@ -30,6 +30,7 @@ package "clock" (
     archive(native) = "mtime_clock.cmxa"
     plugin(byte) = "mtime_clock.cma"
     plugin(native) = "mtime_clock.cmxs"
+    linkopts(javascript) = "+mtime/os/runtime.js"
```

Comment:
@hhugo in ptime you used `.` as separators ( "+ptime.clock.os") rather than `/`, does `jsoo` support both ?

Reply:
jsoo take the part between `+` and the first`/` to figure out the package and ask findlib for the directory.
The part after the first `/` is appended to that directory

Reply:
Ok thanks understood.

Reply:
The version with the `.` is probably more accurate. `"+mtime.clock.os/runtime.js"`

Reply:
Okay don't bother, I'll change that in the reshuffle.